.coveragerc

```diff
@@ -1,6 +1,7 @@
 [run]
 omit = 
     eva/parser/evaql/*
+    eva/udfs/abstract_udf.py
```

Comment:
Why omit the `abstract_udf`?

Reply:
Abstract classes cannot be directly tested so including them in coverage reports is misleading. Adding/removing abstract methods does not actually alter test coverage.